test/basekernels/fbm.jl

```diff
@@ -22,6 +22,6 @@
     @test kernelmatrix(k, x1*ones(1,1), x2*ones(1,1))[1] ≈ k(x1, x2) atol=1e-5
 
     @test repr(k) == "Fractional Brownian Motion Kernel (h = $(h))"
-    test_ADs(FBMKernel, ADs = [:ReverseDiff])
-    @test_broken "Tests failing for kernelmatrix(k, x) for ForwardDiff and Zygote"
+    test_ADs(FBMKernel, ADs = [:ReverseDiff, :Zygote])
+    @test_broken "Tests failing for kernelmatrix(k, x) for ForwardDiff"
```

Comment:
IMO this is a pretty unfortunate and unintended use of `@test_broken` (which was not introduced in this PR, I know). Instead we should just change `test_ADs` in such a way that instead of specifying the tested ADs we state the broken ADs, i.e.,
```suggestion
    test_ADs(FBMKernel, broken = (:ForwardDiff,))
```
Then `test_ADs` would test all AD backends but expect failing tests for the broken backends and tests should pass for all other backends. In this way it is also much easier to keep track of if some backend is actually working or not (might change, e.g., due to changes in the backends without us noticing otherwise).

Reply:
Yes. It is seems more clean way of doing this. However, I think we should address this in the PR where we are revamping AD rules #153 ? Or in a separate PR? 
I hoped for this PR to just get as many AD tests to pass as possible.

Reply:
I guess it might be good to do this in a separate PR before actually switching to ChainRules in #153 (so we clearly know what's the current state). I don't think it should be part of this PR.

Reply:
> IMO this is a pretty unfortunate and unintended use of `@test_broken` (which was not introduced in this PR, I know). Instead we should just change `test_ADs` in such a way that instead of specifying the tested ADs we state the broken ADs, i.e.,
> 
> Then `test_ADs` would test all AD backends but expect failing tests for the broken backends and tests should pass for all other backends. In this way it is also much easier to keep track of if some backend is actually working or not (might change, e.g., due to changes in the backends without us noticing otherwise).

Sorry that's my fault. I just wanted to make sure that we could remember where tests were failing and not forget them :)